### DISCUSSION TITLE
Setting up WSL on a Windows 10 Upgraded to 11. wslregisterdistribution failed with error: 0x80370102

### DISCUSSION THREAD

**Discussion by Ibby-debug** [Cyfrin/foundry-full-course-cu#915] [Q&A]
Whoever runs into this issue where it says you need to enable the virtualization on your computer. You have to enable the virtualization functionality which for some reason could be disabled. 
 
**Solution:**
- hit `shift alt delete`
- Task Manager
-  Performance Tab
- Check if Virtualization is Disabled. 

To turn it on Virtualization, depending on your hardware manufacturer mine was HP. I had to enter the BIOS setup and enabled it from there.

**Comment by cromewar** [comment]
Amazing thanks @IbrahimSam96
